### PR TITLE
Add user settings shortcut

### LINKED ISSUE #1807: User Profile - Shortcut to Settings
### Description (optional, but useful!)
We have identified several opportunities to improve the user profile:

Add a settings button on the profile page so it is easier for a user to get to their settings page
Remove the Activity Feed since it's not actually coming soon
Remove the max-height property on ColonyGrid div (this one was probably done in some other issue)

### Design (optional, might not be applicable)

[Image]User Profile MVP@2x

[Figma link](https://www.figma.com/file/MclWwyYHL04vYVsgKEtgwMaZ/User-Profile?node-id=1524%3A20) - look up for screen labelled as SoT (on the top of the page - user profile 2.0)

### Suggestions (optional)
Pattern is consistent in terms of behaviour and placement to what is on a Colony Home. This request is based on user onboarding insights.

### PR DESCRIPTION
**Pull request by ceolson01**
## Description

This PR adds a shortcut link in the user meta to the user profile edit view.

**Changes** 🏗

* Add settings link to user meta
* Remove user activity (since it's no longer "coming soon")
* Increase spacing between colony grid items per figma spec

**Demo** 🎥 

Profile Shortcut

Closes #1807

### CODE REVIEW

File: src/modules/users/components/UserProfile/UserMeta.css [hunk 1]
```diff
@@ -7,6 +7,13 @@
 
 .headingContainer {
   composes: multiLineEllipsis from '~styles/text.css';
+  display: flex;
+  align-items: center;
+}
+
+.profileLink:nth-child(2) {
+  /* Only add the margni if the link is the second child (adjacent to the user's display name) */
```
Comment (rdig) [inline src/modules/users/components/UserProfile/UserMeta.css]: margin :)

File: src/modules/users/components/UserProfile/UserMeta.tsx [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
+  },
+});
+
 const UserAvatar = HookedUserAvatar({ fetchUser: false });
 
 interface Props {
   user: UserType;
 }
 
+const componentDisplayName = 'users.UserProfile.UserMeta';
```
Comment (rdig) [inline src/modules/users/components/UserProfile/UserMeta.tsx]: :+1:

File: src/modules/users/components/UserProfile/UserProfile.tsx [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 
-import { ActivityFeedItemType, UserType } from '~immutable/index';
+import { UserType } from '~immutable/index';
 import { NOT_FOUND_ROUTE } from '~routes/index';
 import { useDataSubscriber } from '~utils/hooks';
 import { userSubscriber } from '../../subscribers';
 import { useUserAddressFetcher } from '../../hooks';
-import ActivityFeed from '~core/ActivityFeed';
 import ProfileTemplate from '~pages/ProfileTemplate';
-import mockActivities from './__datamocks__/mockActivities';
```
Comment (rdig) [inline src/modules/users/components/UserProfile/UserProfile.tsx]: I think we should remove the mock file as well

File: src/modules/users/components/UserProfile/UserMeta.tsx [hunk 4]
```diff
@@ -33,11 +45,16 @@ const UserMeta = ({
...
-        data-test="userProfileName"
-      />
+      {displayName && (
+        <Heading
+          appearance={{ margin: 'none', size: 'medium', theme: 'dark' }}
+          text={displayName}
+          data-test="userProfileName"
+        />
+      )}
+      <Link className={styles.profileLink} to="/edit-profile">
```
Comment (rdig) [inline src/modules/users/components/UserProfile/UserMeta.tsx]: On my machine the icon is not inlined with the username, although I see in your demo that it is. Can you please double check ?

Screenshot from 2019-10-04 17-07-31

File: src/modules/users/components/UserProfile/UserMeta.css (hunk 1 above)
Comment (ceolson01) [reply src/modules/users/components/UserProfile/UserMeta.css]: Margarita-martinis on the brain 🍹🍸 🤦‍♂

File: src/modules/users/components/UserProfile/UserMeta.tsx (hunk 4 above)
Comment (ceolson01) [reply src/modules/users/components/UserProfile/UserMeta.tsx]: Hmmmm I can't seem to replicate that... 🤔

Cleared cache, tried another machine.... which browser are you using?

**Comment by ceolson01** [comment]
Great, thanks for checking that out again @rdig

**Pull request merged**